Chap_API_Struct.tex

```diff
@@ -882,32 +879,32 @@ \subsection{Job State Structure}
 
```

Comment:
What is a pmix_job_state_t used for?  I don't see anything that uses it other than a pretty print routine.